src/codec_aom.c

```diff
@@ -278,10 +278,9 @@ static avifBool aomCodecEncodeImage(avifCodec * codec,
 
         avifGetPixelFormatInfo(image->yuvFormat, &codec->internal->formatInfo);
 
-        aom_codec_iface_t * encoder_interface = aom_codec_av1_cx();
+        aom_codec_iface_t * encoderInterface = aom_codec_av1_cx();
         struct aom_codec_enc_cfg cfg;
-        aom_codec_enc_config_default(encoder_interface, &cfg, aomUsage);
-        codec->internal->encoderInitialized = AVIF_TRUE;
+        aom_codec_enc_config_default(encoderInterface, &cfg, aomUsage);
```

Comment:
The aom_codec_enc_config_default() call does not initialize the encoder, so I moved the "codec->internal->encoderInitialized = AVIF_TRUE;" after the aom_codec_enc_init() call.